From 7772a57acd0e05353caead7eb7d064e36bcb92e6 Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Sun, 20 Sep 2020 19:32:28 +0800
Subject: [PATCH 34/49] MLK-24519-2 gpu: imx: Increase maximum single pipe
 width to 2560

This patch increase the DPU single pipe maximum from 1920 to 2560 for HDMI/DP.

Signed-off-by: Oliver F. Brown <oliver.brown@nxp.com>
Reviewed-by: Liu Ying <victor.liu@nxp.com>
---
 drivers/gpu/drm/imx/mhdp/cdns-mhdp-imx8qm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/imx/mhdp/cdns-mhdp-imx8qm.c b/drivers/gpu/drm/imx/mhdp/cdns-mhdp-imx8qm.c
index 2ee4e8748b77..cda4d245bab8 100644
--- a/drivers/gpu/drm/imx/mhdp/cdns-mhdp-imx8qm.c
+++ b/drivers/gpu/drm/imx/mhdp/cdns-mhdp-imx8qm.c
@@ -22,7 +22,7 @@
 #define PLL_800MHZ (800000000)
 
 #define HDP_DUAL_MODE_MIN_PCLK_RATE	300000	/* KHz */
-#define HDP_SINGLE_MODE_MAX_WIDTH	1920
+#define HDP_SINGLE_MODE_MAX_WIDTH	2560
 
 #define CSR_PIXEL_LINK_MUX_CTL		0x00
 #define CSR_PIXEL_LINK_MUX_VCP_OFFSET		5
-- 
2.29.2

